### PR TITLE
Add EpisodeExpression for anime file names

### PR DESCRIPTION
**Pull request by TonyBotongChu**
**Changes**
Add an EpisodeExpression to extract episode number from anime style file names like `[VCB-Studio] Re Zero kara Hajimeru Isekai Seikatsu [21][Ma10p_1080p][x265_flac]` and `[CASO&Sumisora][Oda_Nobuna_no_Yabou][04][BDRIP][1920x1080][x264_AAC][7620E503]`.

**Issues**

fixes issue #9332

### CODE REVIEW

File: Emby.Naming/Common/NamingOptions.cs [hunk 1]
```diff
@@ -467,6 +467,14 @@ public NamingOptions()
                 {
                     IsNamed = true
                 },
+
+                // Anime style expression
+                // "[Group][Series Name][21][1080p][FLAC][HASH]"
+                // "[Group] Series Name [04][BDRIP]"
+                new EpisodeExpression(@"(?:\[(?:[^\]]+)\]\s*)?(?<seriesname>\[[^\]]+\]|[^[\]]+)\s*\[(?<epnumber>\d+)\]")
```
Comment (Bond-009) [inline Emby.Naming/Common/NamingOptions.cs:474]: Use `[0-9]` instead of `\d`

**Review by Bond-009 — CHANGES_REQUESTED** [review]

File: Emby.Naming/Common/NamingOptions.cs (hunk 1 above)
Comment (TonyBotongChu) [reply Emby.Naming/Common/NamingOptions.cs]: done

**Review by Bond-009 — APPROVED** [review]

**Review by Shadowghost — APPROVED** [review]

**Comment by TonyBotongChu** [comment]
Hello @Bond-009 and @crobibero , just wanted to kindly check if there’s anything more needed from my side before this PR can be merged. It's already approved, so I thought I'd follow up in case it might have been missed. Thanks!

**Comment by nielsvanvelzen** [comment]
The master branch is currently frozen for the 10.10 release so this PR is on-hold

**Comment by TonyBotongChu** [comment]
> The master branch is currently frozen for the 10.10 release so this PR is on-hold

Thanks for the update! I'll keep an eye on it and will be ready if anything further is needed once the branch is unfrozen.

**Comment by TonyBotongChu** [comment]
Hi team, seems new version has been released for a week, can this PR be merged now?

**Pull request merged**